contracts/libraries/internal/Cron.sol

```diff
@@ -362,6 +362,8 @@ library Cron {
     validateField(spec.minute, MINUTE, 5, 0, 59);
     validateField(spec.hour, HOUR, 1, 0, 23);
     validateField(spec.day, DAY, 1, 1, 31);
+    // TODO: get year
+    // validateField(spec.day, DAY, 1, 1, DateTime.getDaysInMonth(spec.month, year));
```

Comment:
:scissors:

Reply:
How much more gas is it to add this check? Seems worth it.

Reply:
done